bin/csv.js

```diff
@@ -37,6 +37,7 @@ const extractData = (filePath, vendor) => {
       const name = data.name;
       const vendorname = vendorNamesMap.get(vendor) || vendor; // Fallback to the vendor ID if no name is found
       const description = data.description.replace(/"/g, "'");
+      const devicetype = data.devicetype || '';
```

Comment:
Please rename to `deviceType`, lowerCamelCase

Reply:
This applies to what is in the device repository and schema as well

Reply:
Whoops apologies 😁